index.html

```diff
@@ -23,5 +24,36 @@
         </div>
       </div>
     </header>
+    <section class="octopus">
+      <div class="octopus__container container">
+        <div class="octopus__img">
+          <img
+            src="./images/Octopus.png"
+            alt="octopus picture"
+            class="octopus__image"
+          />
+        </div>
+        <h1 class="octopus__title">Обратный звонок на сайт</h1>
+        <h3 class="octopus__subtitle">
+          Получайте в 3 раза больше звонков сразу после установки виджета на Ваш
+          сайт Сервис окупает свой годовой тариф в первую же неделю
+        </h3>
+        <button type="button" class="octopus__button button">ПОДКЛЮЧИТЬ</button>
```

Comment:
```suggestion
        <button type="button" class="octopus__button button">Подключить</button>
```
Please `use text-transform: uppercase; ` in css